.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
```

Comment:
Looks like we should set
```
root = true
```
To indicate this is the root of the project and ensure any higher level configs are not used.